### PR TITLE
Mr 58 product card horizontal landing page and product search

### PR DESCRIPTION
**Pull request by JaqKent**
visual changes on the Horizontal custom card  and fixes

### CODE REVIEW

File: src/components/CustomCard/index.tsx [hunk 1]
```diff
@@ -1,42 +1,90 @@
...
+                        </span>
+                    </div>
+                    {small ? (
+                        <div className={styles.info}>
+                            <span>{onSale}</span>
+                            <span>{delivery}</span>
+                        </div>
+                    ) : (
+                        ''
+                    )}
```
Comment (nathan-hyan) [inline src/components/CustomCard/index.tsx:76]:
```suggestion
                    {small && (
                        <div className={styles.info}>
                            <span>{onSale}</span>
                            <span>{delivery}</span>
                        </div>
                    )}
```

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Great work!

File: src/components/CustomCard/index.tsx [hunk 2]
```diff
@@ -1,42 +1,90 @@
...
+                    </div>
+                    <div className={styles.info}>
+                        {info ? (
+                            <div className={styles.info}>
+                                <span>{onSale}</span>
+                                <span>{delivery}</span>
+                            </div>
+                        ) : (
+                            ''
+                        )}
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/CustomCard/index.tsx:55]: done

File: src/components/CustomCard/index.tsx (hunk 1 above)
Comment (JaqKent) [reply src/components/CustomCard/index.tsx:76]: done

**Pull request merged**